.nf-core-lint.yaml

```diff
@@ -1,3 +1,4 @@
 files_unchanged:
   - lib/NfcoreSchema.groovy
-  - assets/sendmail_template.txt   # Removed MultiQC
+  - assets/sendmail_template.txt # Removed MultiQC
+  - .github/workflows/linting.yml
```

Comment:
why removing this from linting?

Reply:
I will have to dig into this but we don't have `MultiQC` in this pipeline.

Reply:
I agree. Unfortunately, there is no MultiQC for mass spec data. There are some work in progress in different groups but still nothing formal :(